bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -1,2 +1,2 @@
-apiVersion: kubescheduler.config.k8s.io/v1alpha1
```

Comment:
Just a question. Why still `bindata/v4.1.0` and not `bindata/v4.6.0`?

Reply:
No idea, I think it's just to denote that this is v4+. See other components like https://github.com/openshift/cluster-kube-apiserver-operator/tree/master/bindata/v4.1.0

Reply:
Historical, theoretically this could have any name. We initially anticipated that there will a need for upgrade and changing the versions of resource definition based on version, but that was never implemented.